Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3.6.9-slim-stretch
+SHELL ["/bin/bash", "-c"]
+
+RUN mkdir /build
+COPY . /build
+WORKDIR /build
+
+RUN pip install --upgrade pip && pip install -Ur requirements.txt
+
+ENTRYPOINT ["./run.sh"]
```

This should simply be 

```suggestion
ENTRYPOINT ["cellxgene"]
```

so we can parameterize the container from the outside. With the current mechanism we would have to publish one image per instance.